chapters/classes.tex

```diff
@@ -341,8 +341,10 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 \begin{itemize}
 \item
   A variable \lstinline!vc! declared with \lstinline!constant!\indexinline{constant} prefix remains constant during transient analysis, with a value that is unaffected by the initialization problem.
+  This is called a constant, or constant variable.
 \item
   A variable \lstinline!vc! declared with the \lstinline!parameter!\indexinline{parameter} prefix remains constant during transient analysis, with a value determined by the initialization problem.
+  This is called a parameter, or parameter variable.
 \item
```

Comment:
Of course, not just parameter and constant component variabilities should be introduced with `\firstuse` and `\index`.  This comment is a reminder about the other ones.

Reply:
This reminder is no longer needed.  Resolving.